include/pdal/Utils.hpp

```diff
@@ -149,17 +149,17 @@ namespace Utils
 
     inline std::string tolower(const std::string& s)
```

Comment:
The original way led to...

`warning C4244: 'argument' : conversion from 'int' to 'char', possible loss of data`

repeated literally hundreds of times. We could ignore if that's preferred.